var/spack/repos/builtin/packages/libtlx/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libtlx(CMakePackage):
+    """ tlx is a collection of C++ helpers and extensions universally needed,
+        but not found in the STL.
+        The most important design goals and conventions are:
+        1) high modularity with as little dependencies between
+           modules as possible.
+        2) attempt to never break existing interfaces.
+        3) compile on all platforms with C++ – smartphones, supercomputers,
```

Comment:
This dash character needs to be replaced with an ascii dash (likewise with the `-` for `6)`